src/platforms/javascript/common/sourcemaps/uploading/index.mdx

```diff
@@ -6,6 +6,7 @@ notSupported:
   - javascript.nextjs
   - javascript.remix
   - javascript.sveltekit
+  - javascript.astro
```

Comment:
Why do we not want to show this primer intro for Astro?

Reply:
Astro uses Vite out of the box and our SDK package ships with a built-in source map uploading functionality via our Vite plugin. So there's not really a reason to show other uploading methods. I excluded this page because we also exclude it in the other frameworks that have a built-in source map uploading strategy (see the platforms above this line).

Reply:
Do you think we should show this page nevertheless? (happy to discuss but but I'd argue we should be consistent across SDKs here, not just Astro).